Lib/pdb.py

```diff
@@ -1073,6 +1091,34 @@ def _select_frame(self, number):
         self.print_stack_entry(self.stack[self.curindex])
         self.lineno = None
 
+    def do_exceptions(self, arg: str):
```

Comment:
This module is not type annotated, so we shouldn't add this.
My question about the type was: what is "not arg" checking? For None? 0? Empty string?

Reply:
Ah, empty strings, the `def fun(self, arg):` `if not arg` is used a couple of time in the module, but I can add a comment.